components/datepicker/datepicker.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="d-datepicker">
+    <div class="d-datepicker--header">
+      <month-year-picker
+        :prev-month-label="prevMonthLabel"
+        :next-month-label="nextMonthLabel"
+        :prev-year-label="prevYearLabel"
+        :next-year-label="nextYearLabel"
+        :selected-date="selectedDate"
+        @calendar-days="updateCalendarDays"
+      />
+    </div>
+    <div class="d-datepicker--body">
+      <calendar
+        :locale="locale"
+        :calendar-days="calendarDays"
+        :selected-date="selectedDate"
+        @select-date="$emit('selected-date', $event)"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import MonthYearPicker from './modules/month-year-picker.vue';
+import Calendar from './modules/calendar.vue';
+
+export default {
+  name: 'DtDatepicker',
+
+  components: { MonthYearPicker, Calendar },
+
+  props: {
+    prevMonthLabel: {
+      type: String,
+      required: true,
+    },
+
+    nextMonthLabel: {
+      type: String,
+      required: true,
+    },
+
+    prevYearLabel: {
+      type: String,
+      required: true,
+    },
+
+    nextYearLabel: {
+      type: String,
+      required: true,
```

Comment:
```suggestion
    prevMonthLabel: {
      type: String,
      default: 'Previous month',
    },

    nextMonthLabel: {
      type: String,
      default: 'Next month',
    },

    prevYearLabel: {
      type: String,
      default: 'Previous year',
    },

    nextYearLabel: {
      type: String,
      default: 'Next year',
```

Reply:
Try this solution, this will pass the tests and still enable the user to change the labels if needed.

Reply:
Already fixed, it was necessary to declare the props in the storybook

```
// Props
  prevMonthLabel: {
    control: 'text',
    table: {
      type: {
        summary: 'VNode',
      },
    },
  },
  nextMonthLabel: {
    control: 'text',
    table: {
      type: {
        summary: 'VNode',
      },
    },
  },

  prevYearLabel: {
    control: 'text',
    table: {
      type: {
        summary: 'VNode',
      },
    },
  },

  nextYearLabel: {
    control: 'text',
    table: {
      type: {
        summary: 'VNode',
      },
    },
  },
```